packages/sku/config/webpack/webpack.config.js

```diff
@@ -275,14 +275,12 @@ const makeWebpackConfig = ({
         externalizeNodeModules
           ? nodeExternals({
               allowlist: [
-                'classnames', // Workaround for https://github.com/JedWatson/classnames/issues/240
```

Comment:
Do we need to add a changeset for this?

Reply:
Ah, yes we probably do. I guess we need to specify a version upgrade to. I'd like also mention (again) that `Box` support `clsx` in `className` out of the box, and to use `clsx` instead of `classnames`.

Reply:
https://github.com/seek-oss/sku/pull/803/commits/69be3aca9e3bfd7f1efd63b1c2b74538d62084cf